common/common.go

```diff
@@ -21,3 +21,25 @@ type HTTPServerTimeouts struct {
 	Write      time.Duration // Timeout for writes. None if 0.
 	Idle       time.Duration // Timeout to disconnect idle client connections. None if 0.
 }
+
+type BuilderStatus uint8
+
+const (
+	LowPrio BuilderStatus = iota
+	HighPrio
+	Optimistic
+	Blacklisted
+)
+
+func (b BuilderStatus) String() string {
+	switch b {
+	case Optimistic:
+		return "optimistic"
+	case HighPrio:
```

Comment:
May be worth flipping `HighPrio` and `Optimistic` so that the order more closely matches the `const` definition above.

Reply:
done!